include/inexor/vulkan-renderer/meta.hpp.in

```diff
@@ -8,6 +10,7 @@ namespace inexor::vulkan_renderer {
 constexpr const char *APP_NAME{"${INEXOR_APP_NAME}"};
 constexpr std::array<std::uint32_t, 3> APP_VERSION{${INEXOR_APP_VERSION_MAJOR}, ${INEXOR_APP_VERSION_MINOR},
                                                            ${INEXOR_APP_VERSION_PATCH}};
+constexpr std::uint32_t VULKAN_API_VERSION_USED{VK_API_VERSION_1_2};
```

Comment:
I think you forgot to implement the change here?

Reply:
Done.